google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -197,6 +197,10 @@ public static ReadQueryUpdateTransactionOption tag(String name) {
     return new TagOption(name);
   }
 
+  public static ReadQueryUpdateTransactionOption transactionTag(String name) {
```

Comment:
I feel this would bring in confusion to customers, mainly because transaction tags are supported in read-write transactions by using `Options.tag`. Customers might try using this Options.transactionTag for read-write and will not work.
https://cloud.google.com/spanner/docs/introspection/troubleshooting-with-tags#java_1

Should we rename this to be more specific to PDML or is this somehow restricted to be used only for pdml transaction?